cmake/Platform/Linux/PAL_linux.cmake

```diff
@@ -37,3 +37,7 @@ set(LY_ASSET_DEPLOY_ASSET_TYPE "pc" CACHE STRING "Set the asset type for deploym
 
 # Set the python cmd tool
 ly_set(LY_PYTHON_CMD ${CMAKE_CURRENT_SOURCE_DIR}/python/python.sh)
+
+# Set the default window manager that applications should be using on Linux 
+# Note: Only ("xcb", "wayland", or "xlib" should be considered)
+ly_set(PAL_TRAIT_LINUX_WINDOW_MANAGER "xcb")  
```

Comment:
Could the default window manager be placed behind a cache variable?
Like `set(LY_LINUX_WINDOW_MANAGER "xcb" CACHE STRING "Sets the Window Manager to use when configuring Linux")`